OpenRA.Game/Sync.cs

```diff
@@ -112,7 +112,7 @@ public static int HashInt2(int2 i2)
 
 		public static int HashCPos(CPos i2)
 		{
-			return ((i2.X * 5) ^ (i2.Y * 3)) / 4;
+			return i2.Bits;
```

Comment:
This is now no longer the same as `HashInt2` and `HashCVec`, but I couldn't find any reason why these values were chosen to begin with, so this no less "random" as the previous version.